riff-raff/public/docs/howto/configure-a-project.md

```diff
@@ -1,45 +1,25 @@
 Configuring a project
 =====================
 
-To make a project deployable with riff-raff it needs:
+To make a project deployable with Riff-Raff it needs:
  
  - A [`riff-raff.yaml`](../reference/riff-raff.yaml.md) file that describes the deployment process
  - A [`build.json`](../reference/build.json.md) file that details the CI build
- - The two files above along with any assets uploaded into riff-raff's S3 artifact buckets
+ - The two files above along with any assets uploaded into Riff-Raff's S3 artifact buckets
```

Comment:
Worth changing the order here, with the artiact files as the first item as it's the set of files that one controls?

Reply:
https://github.com/guardian/riff-raff/pull/1451/commits/81dcda8f2bf5171ef16d93babbd3a59e4bb1c3c7